CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 - **General:** Introduce new Loki Scaler ([#3699](https://github.com/kedacore/keda/issues/3699))
 - **General**: Add ratelimitting parameters to keda manager to allow override of client defaults ([#3730](https://github.com/kedacore/keda/issues/2920))
 - **General**: Provide Prometheus metric with indication of total number of triggers per trigger type in `ScaledJob`/`ScaledObject`. ([#3663](https://github.com/kedacore/keda/issues/3663))
+- **General:** Introduce new CouchDB Scaler ([#3746](https://github.com/kedacore/keda/issues/3746))
```

Comment:
We have [a style guide for the changelog](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#Changelog), all entries should be alphabetically sorted, with the *General* section in the top (bit also sorted)